manim/mobject/geometry.py

```diff
@@ -2230,9 +2234,11 @@ def construct(self):
 
     def __init__(
         self,
-        color=WHITE,
-        height=2.0,
-        width=4.0,
+        color: Color = WHITE,
+        height: float = 2.0,
+        width: float = 4.0,
+        grid_xstep: float = 0,
```

Comment:
A semantical issue: In the docstring (where typings actually are not necessary, as they are generated automatically when rendering the documentation from the type hints in the function header) the parameter is indicated as "float, optional" -- which means it should be `Optional[float]` here.

This would allow for the default value to be `None` -- which I would actually prefer over 0 (because the spacing between grid lines technically isn't 0 in that case; there *is* no spacing). :-)